axis_backend/apps/documents/serializers/document_serializer.py

```
@@ -0,0 +1,299 @@
+        required=False,
+        allow_null=True,
+        default=list
+    )
+    metadata = serializers.JSONField(required=False, allow_null=True)
+
+    def validate_type(self, value):
+        """Validate document type is valid choice."""
+        if value not in dict(DocumentType.choices):
+            raise serializers.ValidationError(
+                f"Invalid document type. Must be one of: {', '.join(dict(DocumentType.choices).keys())}"
+            )
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)